{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,16 +1,75 @@
 # {{cookiecutter.project_name}}
 
-## Setup
+**_Change this to the full project name 👆_**
 
-- Meet the data science cookiecutter [requirements](http://nestauk.github.io/ds-cookiecutter/quickstart), in brief:
-  - Install: `direnv` and `conda`
-- Run `make install` to configure the development environment:
-  - Setup the conda environment
-  - Configure `pre-commit`
+**_Add a one-liner that describes this project._**
 
-## Contributor guidelines
+## 👋 Welcome!
 
-[Technical and working style guidelines](https://github.com/nestauk/ds-cookiecutter/blob/master/GUIDELINES.md)
+**_Write a longer description of the project here. Put what you like here, but you might want to include:_**
+
+- **_Some background on the project and what its aims are_**
+- **_A high level description of what the code in this repository does_**
+- **_Links to other relevant work or information on Nesta's website_**
+- **_The respository's development status. Is this currently under development, finished, parked (offer an explanation if you feel it is needed)._**
+- **_Link to any particularly important project outputs:_**
+  - **_reports_**
+  - **_slide decks_**
+  - **_media_**
+
+![Maybe add an image or nice graphic relating to the project here. This one is by Alan Warburton / © BBC / Better Images of AI / Plant / CC-BY 4.0](https://betterimagesofai.org/ImagesAI/WarburtonNature_01_1280x720.jpg)
+
+## 📂 Contents
+
+**_Write a high level outline of the repository contents and structure. What are the main functions and outputs and where can the reader find them? This is not a replacement for informative readmes in sub-directories!_**
+
+## 🛠️ Setup and installation
```

Comment:
This section and contributor guidelines section are kind of related (especially because of the github parts of the setup process), so perhaps shouldn't be separated out into two sections/should be next to each other?